create-cardholders-and-cards/server/dotnet/Program.cs

```diff
@@ -1,142 +1,145 @@
-using System.Text.Json;
-using System.Text.Json.Serialization;
+using Microsoft.AspNetCore.StaticFiles.Infrastructure;
 using Microsoft.Extensions.FileProviders;
-
 using Stripe;
 using Stripe.Issuing;
+using CardCreateOptions = Stripe.Issuing.CardCreateOptions;
+using CardService = Stripe.Issuing.CardService;
+
+// For sample support and debugging, not required for production:
+StripeConfiguration.AppInfo = new AppInfo
+{
+    Name = "stripe-samples/issuing/create-cardholder-and-card",
+    Url = "https://github.com/stripe-samples",
+    Version = "0.0.1",
+};
 
 var builder = WebApplication.CreateBuilder(args);
+builder.Configuration.AddEnvironmentVariables();
+
+var stripeKey = builder.Configuration["STRIPE_SECRET_KEY"];
+var webHookSigningSecret = builder.Configuration["STRIPE_WEBHOOK_SECRET"];
+
+builder.Services.AddSingleton(new StripeClient(stripeKey));

```

Comment:
Registering services avoids re-creating them for every request/poll http connection/etc.

Reply:
Ah, I had to set the type for IStripeClient too so that the client was available when the Service is instantiated.

```cs
builder.Services.AddSingleton<IStripeClient>(new StripeClient(key));
```